src/Output/Theme/UnicornTheme.php

```diff
@@ -19,5 +19,7 @@ public function getDefaultColors()
             'info'        => [ CLIColors::$FG_MAGENTA],
             'info_alt'    => [ CLIColors::$FG_WHITE, CLIColors::$BG_MAGENTA ]
         ];
+
+        return array_merge(parent::getDefaultColors(), $styles); // Any styles not defined here, will use the default styles.
```

Comment:
I wonder if we could move this somehow to the CLITheme class so that the child theme classes don't need to do the array_merge thing. What do you think?